bloom-housing/bloom
# refactor(backend): listing translations service

# Pull Request Template

## Issue Overview

This PR addresses #2417 


## Description

This change refactors translations service to take into account new listings schema

## How Can This Be Tested/Reviewed?

Go to public site -> change language to e.g. spanish -> enter a specific listing site -> verify it's correctly translated.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
